docs/user-guide/horizontal-pod-autoscaling/walkthrough.md

```diff
@@ -120,6 +127,145 @@ Here CPU utilization dropped to 0, and so HPA autoscaled the number of replicas
 
 **Note** autoscaling the replicas may take a few minutes.
 
+## Autoscaling on multiple metrics and custom metrics
+
+You can introduce additional metrics to use when autoscaling the `php-apache` Deployment
+by making use of the `autoscaling/v2alpha1` API version.
+
+First, get the YAML of your HorizontalPodAutoscaler in the `autoscaling/v2alpha1` form:
+
+```shell
+$ kubectl get hpa.autoscaling.v2alpha1 -o yaml > /tmp/hpa-v2.yaml
+```
+
+Open the `/tmp/hpa-v2.yaml` file in an editor, and you should see YAML which looks like this:
+
+```yaml
+apiVersion: autoscaling/v2alpha1
+kind: HorizontalPodAutoscaler
+metadata:
+  name: php-apache
+  namespace: default
+spec:
+  scaleTargetRef:
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    name: php-apache
+  minReplicas: 1
+  maxReplicas: 10
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      targetAverageUtilization: 50
+status:
+  observedGeneration: 1
+  lastScaleTime: <some-time>
+  currentReplicas: 1
+  desiredReplicas: 1
+  currentMetrics:
+  - type: Resource
+    resource:
+      name: cpu
+      currentAverageUtilization: 0
+      currentAverageValue: 0
+```
+
+Notice that the `targetCPUUtilizationPercentage` field has been replaced with an array called `metrics`.
+The CPU utilization metric is a *resource metric*, since it is represented as a percentage of a resource
+specified on pod containers.  Notice that you can specify other resource metrics besides CPU.  By default,
+the only other supported resource metric is memory.  These resources do not change names from cluster
+to cluster, and should always be available, as long as Heapster is deployed.
+
+You can also specify resource metrics in terms of direct values, instead of as percentages of the
+requested value.  To do so, use the `targetAverageValue` field insted of the `targetAverageUtilization`
+field.
+
+There are two other types of metrics, both of which are considered *custom metrics*.  These metrics
```

Comment:
If you list the alternative metric types here, you can remove the "first" and "second" phrases below.
There are two other types of metrics, both of which are considered *custom metrics*:
* pod metrics
* object metrics
A pod metric describe a pod...
An object metric describes an object..